dm/tests/shardddl4_1/run.sh

```diff
@@ -1189,7 +1202,7 @@ function DM_RESYNC_NOT_FLUSHED_CASE() {
 	run_sql_source2 "insert into ${shardddl1}.${tb1} values(35,35,35);"
 	run_sql_source2 "insert into ${shardddl1}.${tb2} values(36,36,36);"
 
-	for ((k = 200; k < 240; k++)); do
+	for ((k = 200; k < 280; k++)); do
```

Comment:
can we reduce the DMLs because we now wait the status by checking the log?

Reply:
I tested it in https://github.com/pingcap/tiflow/pull/6578 , seems we don't need this change